{{cookiecutter.repo_name}}/setup.cfg

```diff
@@ -1,2 +1,21 @@
-[wheel]
-universal = 1
\ No newline at end of file
+[mypy]
+ignore_missing_imports = True
+warn_redundant_casts = True
+strict_optional = True
+disallow_untyped_defs = True
+warn_return_any = True
+no_implicit_optional = True
+
+[flake8]
```

Comment:
Does this apply inside pytest? I had this:

[tool:pytest]
flake8-ignore = F811